server/src/account.rs

```diff
@@ -142,6 +111,7 @@ impl Device {
         auth_token: Vec<u8>,
         salt: String,
         registration_id: u32,
```

Comment:
Should this be aci_registration_id instead then?

Reply:
It is just called registrationId on signal server, so I'll just keep it the same